hatch/src/cli/commands/new.rs

```diff
@@ -80,10 +102,11 @@ impl<'command> Command<'command> for New {
       Ok(_) => {
         let version = self.project_version(args);
         let kind = self.project_kind(args);
+        let includes_str =  String::from("  - ") + &self.get_includes(args).join("  - ");
```

Comment:
This produces:
```
  - https://github.com/technetos/Foobar.git  - https://github.com/CachedNerds/Falcon.git%
```
When multiple dependencies are provided

Reply:
A \n got lost some how. I will add it.